src/GDGUkraine/lib/forms/registration_forms.py

```diff
@@ -0,0 +1,268 @@
+        render_kw={
+            'placeholder': 'Start with http://',
+        },
+    )
+    hometown = StringField(
+        label='City',
+        validators=[
+            Required(),
+            Length(max=30),
+        ],
+        render_kw={
+            'placeholder': 'e.g. Mountain View',
+        },
+    )
+    company = StringField(
+        label='Company',
+        validators=[
+            Length(max=64),
+        ],
+        render_kw={
+            'placeholder': 'e.g. Google',
+        },
+    )
+    position = StringField(
+        label='Position',
+        validators=[
+            Length(max=64),
+        ],
+        render_kw={
+            'placeholder': 'e.g. Cofounder',
+        },
+    )
+    experience_level = SelectField(
+        label='Experience level',
+        choices=[(c, c.capitalize()) for c in EXPERIENCE_CHOICES],
+    )
+    experience_desc = TextAreaField(
+        label='Experience description',
+        render_kw={
+            'placeholder': (
+                'e.g. Software developer particularly interested in Java '
+                'technologies and Google applications. Worked in various '
+                'countries: Ukraine, USA, Poland. '
+                'Have great ability to work remotely, high mobility, '
+                'sense of humor.'
+            ),
+        },
+    )
+    interests = TextAreaField(
+        label='Interests',
+        render_kw={
+            'placeholder': 'e.g. Google, Android, Chrome',
+        },
+    )
+    events_visited = StringField(
+        label='Events visited',
+        render_kw={
+            'placeholder': (
+                'e.g. IO Extended 2015, GDG DevFest, hackathons, '
+                'code labs, Study Jams or WTM events'
+            ),
+        },
+    )
+    english_knowledge = SelectField(
+        label='English knowledge',
+        choices=[(c, c.capitalize()) for c in ENGLISH_CHOICES],
+    )
+    t_shirt_size = SelectField(
+        label='T-Shirt size',
+        choices=[(c.lower(), c) for c in TSHIRT_CHOICES],
+    )
+    gender = SelectField(
+        label='Gender',
+        choices=[(c, c.capitalize()) for c in GENDER_CHOICES],
+        validators=[Required()],
+        widget=InlineWidget(prefix_label=False),
+        option_widget=RadioInput(),
+    )
+    comments = TextAreaField(
+        label='Comments',
+        render_kw={
+            'placeholder': 'If you have something to say us, leave it here =)',
+        },
+    )
+
+    def __init__(self, hidden=None, *args, **kwargs):
+        if hidden is None:
+            hidden = []
+
+        # To select default gender:
+        if not kwargs.get('gender'):
+            kwargs.update({'gender': GENDER_CHOICES[0]})
+        super(RegistrationForm, self).__init__(*args, **kwargs)
+
+        for name in hidden:
+            field = getattr(self, name, None)
+            if field and field.flags.required:
+                raise ValueError('Field {} is required'.format(name))
```

Comment:
@webknjaz here it is exactly `ValueError`, as it fails because of invalid value passed to `__init__`.

This code depends on no input, hence it is not the `InputError`. As for `RuntimeError` it is too broad.